### DISCUSSION TITLE
Blog post about DataFusion vs Spark

### DISCUSSION THREAD

**Discussion by alippai** [apache/datafusion#5141] [General]
I searched Google for Spark and DataFusion and this is the first hit: https://www.confessionsofadataguy.com/datafusion-courtesy-of-rust-vs-spark-performance-and-other-thoughts/

Not sure what happened there - was it a missing optimization, bug or something else. I didn’t find any related issues in the issue tracker. Is this something known / already addressed?

**Comment by alippai** [comment]
(I didn’t try the code yet, maybe a —release or a DataFusion upgrade fixes it)

**Comment by dudzicp** [comment]
he  uses datafusion = "11.0"
